src/tests/functional/agenda/test_schedule_export.py

```diff
@@ -62,3 +63,73 @@ def test_feed_view(slot, client, schedule_schema):
     response = client.get(slot.submission.event.urls.feed)
     assert response.status_code == 200
     assert slot.submission.event.schedules.first().version in response.content.decode()
+
+
+@pytest.mark.django_db
+def test_html_export_event_required():
+    from django.core.management import call_command
+    with pytest.raises(CommandError) as excinfo:
+        call_command('export_schedule_html')
+
+    assert 'required' in str(excinfo)
+    assert 'event' in str(excinfo)
+
+
+@pytest.mark.django_db
+def test_html_export_event_unknown():
+    from django.core.management import call_command
+    with pytest.raises(CommandError) as excinfo:
+        call_command('export_schedule_html', 'foobar222')
+
+    assert 'not find event' in str(excinfo)
+    assert 'foobar222' in str(excinfo)
+
+
+@pytest.mark.django_db
+def test_html_export_language(event, slot):
+    from django.core.management import call_command
+    from django.conf import settings
+    import os.path
+
+    event.locale = 'de'
+    event.save()
+    call_command('export_schedule_html', event.slug)
+
+    schedule_html = open(os.path.join(settings.HTMLEXPORT_ROOT, f'test/schedule/index.html')).read()
+    assert 'Kontakt' in schedule_html
+
+
+@pytest.mark.django_db
+def test_html_export(event, other_event, slot, past_slot):
+    from django.core.management import call_command
+    from django.conf import settings
+    import os.path
+
+    call_command('export_schedule_html', event.slug)
+
+    paths = [
+        'static/common/img/logo.svg',
+        'test/schedule/index.html',
+        *[f'test/speaker/{speaker.code}/index.html' for speaker in slot.submission.speakers.all()],
+        f'test/talk/{slot.submission.code}/index.html',
+        *[f'test/speaker/{speaker.code}/index.html' for speaker in past_slot.submission.speakers.all()],
+        f'test/talk/{past_slot.submission.code}/index.html',
+    ]
+
+    for path in paths:
+        full_path = os.path.join(settings.HTMLEXPORT_ROOT, path)
+        assert os.path.exists(full_path)
+
+    assert not os.path.exists(os.path.join(settings.HTMLEXPORT_ROOT, 'test2')), "wrong event exported"
+
+    # views and templates are the same for export and online viewing, so we just need a very basic test here
+    talk_html = open(os.path.join(settings.HTMLEXPORT_ROOT, f'test/talk/{past_slot.submission.code}/index.html')).read()
+    assert talk_html.count('Früher war es nämlich mehr. Und wir mussten es bügeln.') >= 2
+
+    speaker = slot.submission.speakers.all()[0]
+    speaker_html = open(os.path.join(settings.HTMLEXPORT_ROOT, f'test/speaker/{speaker.code}/index.html')).read()
+    assert 'Best speaker in the world.' in speaker_html
+
+    schedule_html = open(os.path.join(settings.HTMLEXPORT_ROOT, f'test/schedule/index.html')).read()
+    assert 'Contact us' in schedule_html  # locale
+    assert 'Lametta im Wandel der Zeiten' in schedule_html
```

Comment:
I'd prefer to use variables for fixture related asserts, so that when we go and change the fixtures, we don't have to fix all tests those fixtures were used in.